src/dev/src/app/radios/radios.demo.html

```diff
@@ -1,65 +1,279 @@
+<!--<label for="compact-radio1" class="clr-control-label">option 1</label>-->
+<!--<input type="radio" id="compact-radio2" name="radio-basic" class="clr-radio">-->
+<!--<label for="compact-radio2" class="clr-control-label">option 2</label>-->
+<!--<input type="radio" id="compact-radio3" name="radio-basic" class="clr-radio">-->
+<!--<label for="compact-radio3" class="clr-control-label">option 3</label>-->
+<!--</div>-->
+<!--&lt;!&ndash; IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER &ndash;&gt;-->
+<!--<clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>-->
+<!--<span class="clr-subtext">Helper Text</span>-->
+<!--</div>-->
+<!--</div>-->
+
+<!--<div class="clr-form-control">-->
+<!--<label class="clr-control-label">-->
+<!--Basic radio, error</label>-->
+<!--<div class="clr-control-container clr-error">-->
+<!--<div class="clr-radio-wrapper">-->
+<!--<input type="radio" name="radio2" id="compact-radio4" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio4" class="clr-control-label">option 1</label>-->
+<!--<input type="radio" name="radio2" id="compact-radio5" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio5" class="clr-control-label">option 2</label>-->
+<!--<input type="radio" name="radio2" id="compact-radio6" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio6" class="clr-control-label">option 3</label>-->
+<!--</div>-->
+<!--&lt;!&ndash; IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER &ndash;&gt;-->
+<!--<clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>-->
+<!--<span class="clr-subtext">Helper Text</span>-->
+<!--</div>-->
+<!--</div>-->
+
+<!--<div class="clr-form-control">-->
+<!--<label class="clr-control-label">-->
+<!--Inline radio</label>-->
+<!--<div class="clr-control-container">-->
+<!--<div class="clr-radio-wrapper clr-radio-inline">-->
+<!--<input type="radio" name="radio3" id="compact-radio7" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio7" class="clr-control-label">option 1</label>-->
+<!--<input type="radio" name="radio3" id="compact-radio8" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio8" class="clr-control-label">option 2</label>-->
+<!--<input type="radio" name="radio3" id="compact-radio9" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio9" class="clr-control-label">option 3</label>-->
+<!--</div>-->
+<!--&lt;!&ndash; IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER &ndash;&gt;-->
+<!--<clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>-->
+<!--<span class="clr-subtext">Helper Text</span>-->
+<!--</div>-->
+<!--</div>-->
+<!--</form>-->
+
+<h2>Angular Radio</h2>
+
+<h4>Vertical</h4>
+
+<label>One <input type="radio" name="one" value="one" [(ngModel)]="vertical.one" /></label>
```

Comment:
Wait, this example doesn't display a Clarity radio button. There is no sibling label after the input, so our styles don't apply at all.

Why is this here? Something we want to support? Some old artifact forgotten here? I'm having a hard time understanding what's happening with this demo file overall.

Reply:
No, this was something I was tinkering with and forgot to remove. This was a bit of a draft to get some input.